### DISCUSSION TITLE
PrusaSlicer now has a coloring feature. It would be great if this worked with PrusaSlicer

### DISCUSSION THREAD

**Discussion by alexqzd** [Sakati84/3DPrintColorizer#65] [Ideas]
[Take a look at this](https://twitter.com/Prusa3D/status/1432626333914472448?s=20)  
Looks very intuitive.

**Comment by Sakati84** [comment]
Thanks for the Info. I will defiantly have a look at the GCode this produces. Won´t promise anything but Cura is getting on my nerves lately...
